beam/docs/zebra_printing.md

```diff
@@ -0,0 +1,178 @@
+# Zebra Printing
+
+To create a Zebra print format, you need the following documents:
+- A ZPL Print Format made against Handling Units that uses the available Jinja utility functions to generate ZPL code.
+- A document Print Format that uses the free Labelary API to convert the above ZPL code and generate a preview of the print output for the linked document.
+
+### ZPL Code Generation
+
+Currently, only three types of printable ZPL data can be generated with utilties within Beam:
```

Comment:
What is the caveat about Graphics fields? Perhaps we should recommend they be developed separately and included as plain text?